common/changes/@autorest/codemodel/internal-consolidate-lint_2021-01-30-01-16.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@autorest/codemodel",
+      "comment": "Internal code linting fixes",
```

Comment:
Would this warrant a release?  Maybe just because the formatting changed and for some reason it could affect behavior?

Reply:
do you mean as an release on its own, I was just thinking of bundling it in the next release